tools/deployment/package-helm/templates/configmap.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: "v1"
+kind: "ConfigMap"
+metadata:
+  name: {{ include "clp.fullname" . }}-config
+  labels:
+    {{- include "clp.labels" . | nindent 4 }}
+data:
+  clp-config.yml: |
```

Comment:
this was partially copied from `var/log/.clp-config.yml` in an initialized clp-package, with modifications to use template variables